askama_derive/src/generator.rs

```diff
@@ -105,12 +104,7 @@ impl<'a> Generator<'a> {
             };
             if path_is_valid {
                 let path = path.to_str().unwrap();
-                buf.writeln(
-                    &quote! {
-                        include_bytes!(#path);
-                    }
-                    .to_string(),
-                )?;
+                buf.writeln(&format!("include_bytes!(\"{path}\");"))?;
```

Comment:
Maybe ↓ incase the name contains backslashes or quotation marks?
```rust
format!("include_bytes!({path:?});")
```

Reply:
I was wondering about it but your solution is just super elegant. Thanks! :)